telemetry/definitions/commonDefinitions.json

```diff
@@ -550,6 +550,36 @@
             "description": "The type of the user trigger to send request to CodeWhisperer service",
             "allowedValues": ["OnDemand", "AutoTrigger"]
         },
+        {
+            "name": "codewhispererFirstCompletionLatency",
+            "type": "double",
+            "description": "The latency from user performs the triggering CW action to plugin call GenerateCompletions API for the first recommendation"
+        },
+        {
+            "name": "codewhispererEndToEndLatency",
+            "type": "double",
+            "description": "The latency from plugin start invocation to plugin call GenerateCompletions API to first completion shown in the IDE"
+        },
+        {
+            "name": "codewhispererAllCompletionsLatency",
+            "type": "double",
+            "description": "The latency from plugin makes a first call to GenerateCompletions API to receiving the response for the last call to the GenerateCompletions API."
```

Comment:
The time it takes for the last GenerateCompletions response to be received after plugin makes a first call to GenerateCompletions API